pysmt/smtlib/parser.py

```diff
@@ -32,6 +33,15 @@
 from pysmt.utils import interactive_char_iterator
 
 
+def open_(fname):
+    """Transparently handle .bz2 files."""
+    if fname.endswith(".bz2"):
+        if PY2:
+            return bz2.BZ2File(fname, "r")
+        else:
+            return bz2.BZ2File(fname, "rt")
```

Comment:
This seems to fail in Python 3.4... (see Travis)

Maybe we need a better distinction on the python version and put this logic in utils?